### ISSUE TITLE
JAWS is not announced aria-selected=false in multi-select component

### ISSUE THREAD

**Issue by glushkova91** [FreedomScientific/VFO-standards-support#159]
### Summary

1. Go to https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-rearrangeable.html#ex2_label
1. Enable JAWS SR and scroll down to "Example 2: Multi-Select Listbox"
2. Select any item from the list, check the result: SR announces that item is selected
3. Unselect this item - check the result: SR doesn't announce that item is not selected

### Expected result

When you unselect an item from multi-select list SR should announce that item is Not Selected now.

### Actual result

SR doesn't announce unselected status.

### Example

https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-rearrangeable.html#ex2_label

#### JAWS version and build number
JAWS Version 2018.1803.24 
 
#### Operating System and version
Windows 10 Enterprise (64-bit)

#### Browser and version:
Google Chrome

**Comment by jducrot** [comment]
By Default, JAWS only announces "Not Selected". You can change that in the settings. In JAWS 20, I was able to update the setting by going to "Settings Center > Text Processing > General > List Item" and choosing between Say None, Say Both, Say Selected, and Say Unselected (default).

![image](https://user-images.githubusercontent.com/280674/82606686-7c80e880-9b85-11ea-88ae-fb494374a06c.png)
